jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -87,7 +87,7 @@ YUI.add('deployment-flow', function() {
         deploying: false,
         credential: this.props.credential,
         loadingTerms: false,
-        loggedIn: !!this.props.getAuth(),
+        loggedIn: !!this.props.isLoggedIn(),
```

Comment:
`isLoggedIn` returns a boolean so the !! is not necessary.

before `getAuth` returned a non-boolean value so !! would type coerce